Makefile

```diff
@@ -1,36 +1,49 @@
 # Variables required for this Makefile
-ROOT_DIR = $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
-VERSION = $(shell git describe --tags --always)
-GO_ENV_VARS =
-
-ifdef GOOS
-GO_ENV_VARS = GOOS=$(GOOS)
-endif
-
-ifdef GOARCH
-GO_ENV_VARS += GOARCH=$(GOARCH)
-endif
-
-DOCKER_MULTI_ARCH_PLATFORMS = linux/amd64,linux/arm64
+# Include all common variables
+include Makefile.vars
 
 # Builds exporter binary
 .PHONY: exporter
 exporter:
-	$(GO_ENV_VARS) go build -ldflags="-X 'main.version=$(VERSION)'" -o aerospike-prometheus-exporter .
+	@echo $(GO_FIPS)
+	$(GO_ENV_VARS) GOEXPERIMENT=$(GO_FIPS) go build -ldflags="-X 'main.version=$(VERSION)'" -o aerospike-prometheus-exporter .
+
+.PHONY: fipsparam
+fipsparam: 
+ifeq ($(APE_SUPPORTED_OS),validfipsos)
```

Comment:
I think this block can be simplified more. 
But lets not change this in the last minute.
We will polish it as part of future changes.